deployment/document-understanding-solution.template

```diff
@@ -21,7 +21,14 @@ Parameters:
   KendraEnabled:
     Description: Amazon Kendra is added as an additional capability and can be used for exploring features such as Semantic Search, Adding FAQs and Access Control Lists.
```

Comment:
I know this was out of the scope of your PR, but could we add a statement. This mode comes pre-loaded with a sample dataset of medical documents. ?The reason I ask is because then the user may want to enable Comprehend Medical as well

Reply:
Sure I can add it.